packages/apps/plugins/plugin-stack/src/components/StackMain.tsx

```diff
@@ -110,7 +109,7 @@ const StackMainImpl = ({ sections }: { sections: StackSections }) => {
     if (active.id !== over?.id) {
       const oldIndex = sections.findIndex((section) => section.object.id === active.id);
       const newIndex = sections.findIndex((section) => section.object.id === over?.id);
-      arrayMoveInPlace<StackSectionModel>(sections, oldIndex, newIndex);
+      arrayMove(sections, oldIndex, newIndex);
```

Comment:
Does this perform an in-place move? It’s just that this shares the name of `dnd-kit`’s function for this which returns a new array instead.